README.md

```diff
@@ -10,7 +10,23 @@ First, you'll need to configure a database. Once this project is deployed, we'll
 
 ### Windows setup
 
-Coming Soon!
+## Chocolatey and Postgres (Needs Testing)
+1. [Install Chocolatey](https://chocolatey.org/docs/installation#installing-chocolatey) if you do not already have it installed.
+2. Add `C:\ProgramData\chocolatey\bin` to your paths (type path in search, click Edit environment varaibles for you account).
+3. In your powershell (admin), run `choco install postgresql10` (powershell (admin) can be found by pressing `win + x`).
+4. Upgrade postgresql: `choco upgrade postgresql`.
+5. Add `C:\Program Files\PostgreSQL\10\bin` to your [paths](https://helpdeskgeek.com/windows-10/add-windows-path-environment-variable/).
+6. Restart your admin terminal to update the path.
```

Comment:
I don't like having them restart the terminal because of the path, and I also don't like having to open up the window where environment variables are stored twice.

Can we just have them add both things to the path in one step before the powershell session is ever started, then it's already correct once they actually install it? (Not sure if Windows would complain about this, I know this would be fine on Linux)

Reply:
I thought about that. I do not /think/ it would cause complaints. I left it separate for two reasons: 1. unsure about the complaints, 2. I added to Development Notes to see step 22 for the FLASK_APP=run.py.